gh_actions_exporter/metrics.py

```diff
@@ -74,14 +95,39 @@ def set_status(self, webhook: WebHook) -> str:
             status=webhook.workflow_run.status,
             repo=webhook.repository.full_name,
             created_at=webhook.workflow_run.created_at,
+            finished_at=finished_at
         ).state(status)
         return status
 
     def cleaner(self):
         def test(elem):
```

Comment:
be more clear in the naming of this function please

Reply:
which function? `cleaner` or `test`?

Reply:
Both, what are your testing ? 
If i understand well, this function take the a build and remove it if it status is completed 5 min ago.
So it could be called `is_build_completed` with a small doc underneath it.